test-suite-helper/src/main/java/io/micronaut/testsuitehelper/TestGeneratingAnnotationProcessor.java

```diff
@@ -37,7 +37,7 @@
 @SupportedSourceVersion(RELEASE_17)
 public class TestGeneratingAnnotationProcessor extends AbstractProcessor {
 
-    private boolean executed = false;
+    private boolean executed;
```

Comment:
There is no concrete difference between these two lines I don't want 
to be overly nit picky but what are the reasons for this change?

Reply:
@chalkedgoose It makes no sense to assign default values to primitives. You don't write an assignment next to every attribute of the class, so why should you assign a default value here? Anyway, it's just a smell code correction.

Reply:
i disagree with the linter, i think in this case `= false` is more clear, it shows the value is initially false and not e.g. set in a constructor. Though it's not a big deal.

Reply:
@yawkat Well, then we need to decide: either we rely on the knowledge of java, or we are against abbreviations and write always default values, so that the code is "cleaner and understandable". That is, you need to correct in the other direction, classes, like:
```java
class MyClass {

    private String f1;
    private float f2;
    private Double f3;
    private int f4;
...
}
```
It should be described as follows:
```java
class MyClass {

    private String f1 = null;
    private float f2 = 0F;
    private Double f3 = null;
    private int f4 = 0;
...
}
```
It's getting really weird right now. Everyone knows that the default boolean is false, and in most cases the default value is not written, and it is written in this place. So what's the difference between this place and the others?

Reply:
the difference is entirely subjective. in some places it's more clear to have an explicit initializer, in others it is not.

Reply:
I will just say that everyone decides for themselves what is more convenient for them. I still attribute this change to the code style. If you think this is not a necessary change, no problem, I don't insist ;-)